integrations/integration_test.go

```diff
@@ -155,21 +156,19 @@ func (s *TestSession) MakeRequest(t *testing.T, req *http.Request) *TestResponse
 	return resp
 }
 
-func loginUser(t *testing.T, userName, password string) *TestSession {
```

Comment:
No, I don't think we should do that. How should we test user login with different password?